frontend/src/pages/requests/helpers.tsx

```diff
@@ -185,14 +185,13 @@ export const PatientsListColumns: ColDef[] = [
     sortable: false,
   },
   {
-    field: "primaryIds",
-    headerName: "Primary IDs",
+    headerName: "CMO Sample IDs",
     valueGetter: function ({ data }) {
-      let sampleIds = [];
-      for (let sample of data["isAliasPatients"][0].hasSampleSamples) {
-        sampleIds.push(sample.hasMetadataSampleMetadata[0].primaryId);
-      }
-      return sampleIds.join(", ");
+      return data["isAliasPatients"][0].hasSampleSamples.map(
+        (sample: Sample) =>
+          sample.hasMetadataSampleMetadata[0].cmoSampleName ||
+          sample.hasMetadataSampleMetadata[0].primaryId
+      );
```

Comment:
Quick note on the OR operator: (`||`)
If the first operand is truthy (e.g. not undefined), then the OR operator will short-circuit and return the first operand without evaluating the second operand. Otherwise, it will return the second operand. Example:
```
true || "hello"  // returns true
false || "hello" // returns "hello"
```